.gitignore

```diff
@@ -1,5 +1,12 @@
-/composer.lock
 /vendor/
+/node_modules/
+/composer.lock
+
 /etc/build/*
-!/etc/build/.gitkeep
-.idea/
```

Comment:
I would like to keep `.idea/`, have seen getting that one committed a bit too often in the meanwhile.

Reply:
Not a very big fan of having IDE specific things in repository. I think this is a matter of disciplining contributors :D If you have look at the main part of big repositories you won't see IDE specific files in there.